### DISCUSSION TITLE
Moving away from Python scripts

### DISCUSSION THREAD

**Discussion by nbolton** [deskflow/deskflow#7550] [Development]
The Python scripts in the `scripts` dir are useful for some developers but are also seen as an inconvenience to some developers, increasing the entry barrier for some new contributors. In some cases the Python scripts are a crutch caused by a lack of knowledge on CMake and other tools. On some systems, such as FreeBSD, these scripts can be an outright impediment, which is why we created the `install_deps.sh` script which side-steps Python entirely on FreeBSD.

We should move away from needing to run Python scripts by improving our CMake configuration and relying more on `vcpkg` (on Windows). There will be some things that the Python scripts will be useful for, so we shouldn't abandon them entirely, but we should ensure that new contributors don't have to use them if they cannot or do not want to.

**Comment by sithlord48** [comment]
Closing as this is completed
